README.md

```diff
@@ -1,69 +1,105 @@
-# Project "Piper" Overview
+# Description
 
-An efficient software development process is vital for success in building business applications on SAP Cloud Platform or SAP on-premise platforms.
-SAP addresses this need for efficiency with project "Piper". The goal of project "Piper" is to substantially ease setting up continuous deployment processes for the most important SAP technologies by means of Jenkins pipelines.
+An efficient software development process is vital for success in building
+business applications on SAP Cloud Platform or SAP on-premise platforms.
+SAP addresses this need for efficiency with project "Piper". The goal of project
+"Piper" is to substantially ease setting up continuous deployment processes for
+the most important SAP technologies by means of Jenkins pipelines.
 
 Project "Piper" consists of two parts:
 
- * A [shared library][piper-library] containing steps and utilities that are required by Jenkins pipelines.
- * A set of [Jenkins pipelines][piper-pipelines] using the piper library to implement best practice processes.
+ * A [shared library][piper-library] containing steps and utilities that are
+   required by Jenkins pipelines.
+ * A set of [Jenkins pipelines][piper-pipelines] using the piper library to
+   implement best practice processes.
 
 ## What you get
 
-This repository contains Jenkins pipeline instructions in form of a `Jenkinsfile` (together with some auxiliary files) which require the [shared library][piper-library] of project "Piper" to run. They are available under `pipelines/<scenario>/Jenkinsfile` in this repository.
+This repository contains Jenkins pipeline instructions in form of a
+`Jenkinsfile` (together with some auxiliary files) which require the [shared
+library][piper-library] of project "Piper" to run. They are available under
+`pipelines/<scenario>/Jenkinsfile` in this repository.
 
-These best practice Jenkinsfiles are based on the general concepts of [Pipelines as Code with Jenkins][jenkins-doc-pipelines]. They give you the power of the Jenkins community at hand to optimize your continuous delivery processes.
+These best practice Jenkinsfiles are based on the general concepts of [Pipelines
+as Code with Jenkins][jenkins-doc-pipelines]. They give you the power of the
+Jenkins community at hand to optimize your continuous delivery processes.
 
-You can run the Jenkinsfiles out of the box or take them as a starting point for project-specific adaptations.
+You can run the Jenkinsfiles out of the box or take them as a starting point for
+project-specific adaptations.
 
-## Installation
+# Requirements
 
-Prerequisites:
-
-* [Installed shared library][piper-library-installation] of project "Piper".
+* [Installed shared library][piper-library-readme] of project "Piper".
```

Comment:
How about linking to the installation section of the readme here.
https://github.com/SAP/jenkins-library/blob/master/README.md#download-and-installation

Reply:
Done